cvat/schema.yml

```diff
@@ -8888,6 +9040,31 @@ components:
         * `natural` - NATURAL
         * `predefined` - PREDEFINED
         * `random` - RANDOM
+    StatisticsItem:
+      type: object
+      properties:
+        title:
+          type: string
+        description:
+          type: string
+        granularity:
+          $ref: '#/components/schemas/GranularityEnum'
```

Comment:
Probably, these enum fields should allow a null value instead of having a dedicated enum member for this. I'm not sure if none is a valid member for such enum.